docs/api-designer/getting-started-api-designer/README.adoc

```diff
@@ -101,147 +108,142 @@ In this quick start, you'll learn how to ...
 endif::[]
 
 [id="proc-creating-schema-api-design_{context}"]
-== Creating an {product-api-designer} schema or API design
+== Creating a schema or API design with {product-long-api-designer}
 
 [role="_abstract"]
-Use the {product-long-api-designer} web console to create an {product-api-designer} schema or API design.
+Use the {product-long-api-designer} web console to create an event schema or API definition.
 
 ifndef::qs[]
 .Prerequisites
 * You're logged in to the {product-api-designer} web console at {service-url-api-designer}[^].
 endif::[]
 
 .Procedure
-. In the {service-url-api-designer}[{product-api-designer} web console], click ...
-. Click ...
+. In the {service-url-api-designer}[{product-api-designer} web console], click *Create a schema or API design*.
+. Complete the form to provide the details of the new design:
+.. *Name*: Enter a unique design name, such as `my-api-design`.
+.. *Summary*: Optional: Enter a short description of the design.
+.. *Type*: Accept the default value: *OpenAPI*.
+.. *Version*: Accept the default value: *3.0.2*.
+.. *Template*: Select the *Pet Store Example* template.
```

Comment:
Is OpenAPI the use case that PM want to show (i.e., not Kafka-based example with Avro)?

Reply:
Yes, upstream Apicurio Studio also has OpenAPI default, so probably going with that for now. Will probably add templates for other types later.

Reply:
Added an Avro example in latest commit :)